wfe/wfe.go

```diff
@@ -712,6 +731,109 @@ func (wfe *WebFrontEndImpl) UpdateAccount(
 	}
 }
 
+func (wfe *WebFrontEndImpl) verifyKeyRollover(
+	innerPayload []byte,
+	existingAcct *core.Account,
+	newKey *jose.JSONWebKey,
+	request *http.Request) *acme.ProblemDetails {
+	var innerContent struct {
+		Account string
+		OldKey  jose.JSONWebKey
+	}
+	err := json.Unmarshal([]byte(innerPayload), &innerContent)
```

Comment:
I don't think you need the `[]byte()` cast on `innerPayload` since its already a `[]byte` argument

Reply:
Yes, it also works without it.